lib/src/id_prefix.rs

```diff
@@ -32,7 +32,7 @@ struct DisambiguationData {
 }
 
 struct Indexes {
-    commit_index: IdIndex<CommitId, CommitId>,
+    commit_index: IdIndex<CommitId, ()>,
```

Comment:
The (poorly placed) TODO on line 30 can go away now

Reply:
Right, removed the TODO comment.